public/index.html

```diff
@@ -268,6 +268,24 @@
             <div class="project-name">Budget Tracker</div>
             <a href="Budget Tracker/index.html" class="demo-link">Demo</a>
         </div>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 47</div>
+            <div class="project-name">Budget Tracker</div>
+            <a href="Budget Tracker/index.html" class="demo-link">Demo</a>
+        </div>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 47</div>
+            <div class="project-name">Budget Tracker</div>
+            <a href="Budget Tracker/index.html" class="demo-link">Demo</a>
+        </div>
+    </div>
+        <div class="table">
```

Comment:
change day 
make index html properly
PULL main branch first 
budget tracker is showing 3 times